### ISSUE TITLE
Eyelids scaling

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9807]
**Preferred term label:**
Eyelids scaling

**Synonyms**
Collarettes

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Blepharitis, infectious

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Eyelind scaling and Collarettes (see https://pubmed.ncbi.nlm.nih.gov/35449733/) are separate findings? Should we code Collarette?

**Comment by MickeySegal** [comment]
They do seem only vaguely related.  Let's just call this finding Collarettes.

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000421
- label: Collarettes
- parents: [Abnormal eyelid morphology (HP:0000492)](https://hpo.jax.org/app/browse/term/HP:0000492)
- pmid: 
- def: Buildup of waxy flesh-colored scales on the margin of the eyelid.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9807
- synonyms: (NONE)
